_includes/setup/system-linux.md

```diff
@@ -0,0 +1,8 @@
+## System requirements
```

Comment:
These aren't in the sidenav. Does our sidenav support 3 levels in the hierarchy?